### ISSUE TITLE
Golangci-lint v1.17.1 docker build failure on OdroidHC2

### ISSUE THREAD

**Issue by Adminiuga** [pusher/oauth2_proxy#417]
Trying to build a docker image (current master) on Odroid HC2 (XU4) running Armbian Linux (Debian Buster) on 4.14.150-odroidxu4 kernel with `Docker version 19.03.5, build 633a0ea` docker.

Building fails with:
```
Step 1/14 : FROM golang:1.13-buster AS builder
 ---> 1a37b9efe7b6
Step 2/14 : RUN curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.17.1
 ---> Running in 3d99c1fda6d9
golangci/golangci-lint info checking GitHub for tag 'v1.17.1'
golangci/golangci-lint info found version: 1.17.1 for v1.17.1/linux/armv7
The command '/bin/sh -c curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.17.1' returned a non-zero code: 1
```

Bumping up golangci-lint version to 1.18.0 or 1.23.6 makes this problem go away

## Expected Behavior
Build to finish.

## Current Behavior
Building fails with 
```
Step 1/14 : FROM golang:1.13-buster AS builder
 ---> 1a37b9efe7b6
Step 2/14 : RUN curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.17.1
 ---> Running in 3d99c1fda6d9
golangci/golangci-lint info checking GitHub for tag 'v1.17.1'
golangci/golangci-lint info found version: 1.17.1 for v1.17.1/linux/armv7
The command '/bin/sh -c curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.17.1' returned a non-zero code: 1
```

## Possible Solution
Bump `golangci-lint` version to `1.23.6`

## Steps to Reproduce (for bugs)

1.  Clone/checkout oauth2_proxy repo
2.  `docker build - < Dockerfile.armv6`

## Your Environment
Odroid HC2 (xu4) SBC running Armbian LInux (essentially Debian Buster with odroid kernel)
```
$ uname -a
Linux hc2 4.14.150-odroidxu4 #2 SMP PREEMPT Mon Oct 28 08:07:45 CET 2019 armv7l GNU/Linux
```

- Version used: Current "master" tree or 5.0.0

**Comment by JoelSpeed** [comment]
Looks like we can just bump the version being installed across all of our dokcerfiles and this should be fixed? Anyone got time to raise a PR for this?

**Comment by JoelSpeed** [comment]
Looks like this will be fixed by #419

**Comment by Adminiuga** [comment]
Thank you. Closing this one for now and waiting for #419 to go through.
